packages/release-mastermind/src/conditions/util/isAbandoned.ts

```diff
@@ -0,0 +1,24 @@
+import { UtilProps, UtilThis } from "..";
+
+const TYPE = "isAbandoned";
+
+export interface ConditionIsAbandoned {
+  type: typeof TYPE;
+  value: number;
+  label: string;
+}
+
+function isAbandoned(
+  this: UtilThis,
+  condition: ConditionIsAbandoned,
+  issue: UtilProps
+) {
+  if (!issue.lastUpdated || !issue.labels?.[condition.label.toLowerCase()])
```

Comment:
Codacy has a fix for the issue: [Unexpected nullable string value in conditional. Please handle the nullish/empty cases explicitly.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
  if ((issue.lastUpdated == null) || !issue.labels?.[condition.label.toLowerCase()])
```